verification/curator-service/ui/src/components/Users.tsx

```diff
@@ -36,14 +69,68 @@ export default class Users extends React.Component<{}, UsersState> {
             });
     }
 
+    updateRoles(event: React.ChangeEvent<{ value: unknown }>, updatedUser: User): Promise<unknown> {
```

Comment:
It can be - it requires casting the event object down below in the onChange method. But it allows for removing a cast within this method, so probably a net positive. Done.